package main

func test02(a, b int) int {
	return a + b
}
func main() {
	panic("系统错误")
	test02(1, 2)
}
